routes/get-routes.js

```diff
@@ -7,15 +7,13 @@ module.exports = (app) => {
     passport.authenticate("jwt", { session: false }),
     async (req, res) => {
       try {
-         console.log("here in existing new profile ", req.user)
         const existingProfile = await profile.getProfileInfo(req.user.userId);
```

Comment:
i actually don't like that 'userId'. It is an email and I think it should be renamed in the model/db/everywhere. Could be a separate task to refactor a model

Reply:
Yes, let's make it in a separate task